google/cloud/bigquery/v2/minimal/internal/job_logging.cc

```diff
@@ -46,6 +50,10 @@ Result JobLogWrapper(Functor&& functor, rest_internal::RestContext& context,
   auto response = functor(context, request);
   GCP_LOG(DEBUG) << where << "() >> status=" << response.status();
 
+  if (response.ok()) {
+    GCP_LOG(DEBUG) << ", " << response->DebugString(response_name, options);
+  }
```

Comment:
`LogWrapper` renders such cases as ...
```
  if (!response) {
    GCP_LOG(DEBUG) << where << "() >> status="
                   << DebugString(response.status(), options);
  } else {
    GCP_LOG(DEBUG) << where
                   << "() >> response=" << DebugString(*response, options);
  }
```
`JobLogWrapper` should follow the same pattern.